### PR TITLE
Updated yelling.py

### PR DESCRIPTION
**Pull request by paulrusthewalrus**
This change should stop the bot from chastising you just because you posted a link in #yelling.

### CODE REVIEW

**Comment by nicklambourne** [comment]
Approach is a bit simple, maybe try a regex match instead? 
e.g. https://stackoverflow.com/a/3809435

**Comment by paulrusthewalrus** [comment]
Whoops, didn't see your message before I made this new PR. Will try that now :)

**Comment by kentonlam** [comment]
This closes #534.

File: uqcsbot/scripts/yelling.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 from random import choice, random
 from re import sub, UNICODE
+import re
```
Comment (nicklambourne) [inline uqcsbot/scripts/yelling.py:5]: `import re` should go first in imports because it's a global import (the whole package).

**Review by nicklambourne — CHANGES_REQUESTED** [review]
First, have you tested this in the test workspace? Can you post some screenshots of it working? Will also need a test or two at some point.
Second, I don't think with the current setup that it will work with this one:

> f"DID YOU SAY \n>>>{mutate_minuscule(text)}".upper(),

You either need to put the link back in somehow or remove this from the available options if a link exists.

**Comment by bradleysigma** [comment]
Odd things may happen if the bot chooses the `f"DID YOU SAY \n>>>{mutate_minuscule(text)}".upper()` response when there is both a link and other, non-link minuscule characters in the message.

**Comment by paulrusthewalrus** [comment]
Tests for the new PR:
Screen Shot 2020-10-03 at 5 34 07 pm

**Comment by nicklambourne** [comment]
Might I suggest using re.match() to identify where the links were so you can reinsert them after analysis and mutation?

**Comment by paulrusthewalrus** [comment]
Screen Shot 2020-11-19 at 3 18 06 pm

**Review by nicklambourne — APPROVED** [review]

**Comment by nicklambourne** [comment]
@kentonlam @TomCranitch Can one of you two have a look at this? I changed a little more than I expected to in the cleanup.

**Review by TomCranitch — APPROVED** [review]

**Pull request merged**